### DISCUSSION TITLE
Guys, I think you've lost the one version. Where's 26.0.3? 🤣

### DISCUSSION THREAD

**Discussion by alexted** [keycloak/keycloak#34498] [Q&A] [answered]
Once again you release a version that doesn't match the next sequence number. Is this some kind of trick?
Where's the `26.0.3` gone??

**Comment by jonkoops** [comment, marked as answer]
There was a GitHub outage that caused the release for 26.0.3 to end up in an unrecoverable state, so we decided to roll out 26.0.4 as a replacement. You can safely skip 26.0.3.
